src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = false;
```

Comment:
I suggest moving setting default health value into constructor's parameters and write something like that: constructor(name, health = 100).

Reply:
I don't think it is necessary because health is always  starts from 100